microshift.sh

```diff
@@ -70,6 +70,8 @@ function download_microshift_rpm {
     local extra_opts=""
     local nvr_suffix=""
     if [ -n "${MICROSHIFT_PRERELEASE-}" ]; then
+        # Need to set this setting to repo level so that each run of yum command able to consume it instead only the `yum download`
+        sudo yum config-manager --save --setopt=rhocp-${OPENSHIFT_MINOR_VERSION}-for-rhel-9-$(uname -i)-rpms.skip_if_unavailable=true
```

Comment:
Why enable `rhocp-${OPENSHIFT_MINOR_VERSION}-for-rhel-9-$(uname -i)-rpms` at all in `enable_repos` when `MICROSHIFT_PRERELEASE` is set if we know it's not going to exist?

Reply:
As soon as it hit to RC, they usually appear. But yes I had different thought around it ( might be some dependency package part of this repo) but we can remove it and and the flag at repo enable part.

Reply:
> As soon as it hit to RC, they usually appear. But yes I had different thought around it

This is not mentioned in the commit log, I cannot guess this during review